flow-bridge/src/main/java/org/reactivestreams/ReactiveStreamsFlowBridge.java

```diff
@@ -110,13 +110,39 @@ private ReactiveStreamsFlowBridge() {
         }
         return new FlowToReactiveProcessor<T, U>(reactiveStreamsProcessor);
     }
-    
+
+    /**
+     * Converts a Reactive Streams Subscriber into a Flow Subscriber.
+     * @param <T> the input and output value type
+     * @param reactiveStreamsSubscriber the Reactive Streams Subscriber instance to convert
+     * @return the equivalent Flow Subscriber
+     */
+    public static <T> Flow.Subscriber<T> toFlow(org.reactivestreams.Subscriber<T> reactiveStreamsSubscriber) {
```

Comment:
I cherry picked https://github.com/reactive-streams/reactive-streams-jvm/pull/399 assuming we'll merge that addition before this PR though

Reply:
We can adjust naming after merging #399 as is.

Reply:
Yeap, I don't mind merging it as-is :) esp since this will want to base on it